lib/pymedphys/docs/lib/tutes/gamma/gamma_tutorial.ipynb

```diff
@@ -0,0 +1,571 @@
+{
```

Comment:
<p>Maybe a good point to flesh out why this is the case.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymedphys/pymedphys/pull/1638/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I can't see the cell you refer to in this comment.

Reply:
@nlambriICH if you click on the following link:

![image](https://user-images.githubusercontent.com/6559099/172263627-aa2a6643-07aa-42b7-ba36-ad43ca3c1896.png)

and then scroll down to this section of the review:

![image](https://user-images.githubusercontent.com/6559099/172263566-3165e9dc-29fe-4652-b6bf-644e70c962ac.png)

Hopefully that helps?

Reply:
(You might have to log in with your GitHub account first)